src/ulid.cr

```diff
@@ -19,6 +20,49 @@ module ULID
     encode_time(seed_time, TIME_LEN) + encode_random(RANDOM_LEN)
   end
 
+  # Validate a string is a ULID
+  #
+  # ```
+  # ULID.valid?("01B3EAF48P97R8MP9WS6MHDTZ3")
+  # # => true
+  # ```
+  def valid?(ulid : String) : Bool
+    # Incorrect length && Invalid chars
+    ulid.size == TIME_LEN + RANDOM_LEN && ulid.upcase.chars.all? &.in?(ENCODING)
+  end
+
+  # Validate a string is a ULID
+  #
+  # ```
+  # ULID.valid!("01B3EAF48P97R8MP9WS6MHDTZ3")
+  # # => nil (or exception)
+  # ```
+  def valid!(ulid : String) : Nil
+    # Incorrect length
+    raise IncorrectLength.new("ULIDs must be 26 characters.") unless ulid.size == TIME_LEN + RANDOM_LEN
+    # Invalid chars
+    raise InvalidChars.new("Invalid characters found.") unless ulid.upcase.chars.all? &.in?(ENCODING)
+
+    return nil
```

Comment:
no need `return` here as it always return the last value

Reply:
if you install ameba in your bin folder or install ameba extension in vscode, the linter will pick this up for you